src/components/ComponentSettingsModal.jsx

```diff
@@ -181,7 +182,7 @@ const ComponentSettingsModal = ({
                 setFileUrl(frontMatter.file_url)
                 setOriginalFileUrl(frontMatter.file_url)
 
-                setResourceDate(frontMatter.date)
+                setResourceDate(type === 'resource' ? retrieveResourceFileMetadata(fileName).date : frontMatter.date)
```

Comment:
Are there any cases other than resource pages where the `date` attribute in the front matter is used?

Reply:
You're right, there aren't any others - I've removed this from other cases in https://github.com/isomerpages/isomercms-frontend/pull/274/commits/56280f359168d893548e6547c92be6f292e71566